vyper/ir/compile_ir.py

```diff
@@ -837,6 +838,31 @@ def _prune_inefficient_jumps(assembly):
     return changed
 
 
+def _optimize_inefficient_jumps(assembly):
```

Comment:
let's leave this as a separate PR (there are two, duplicate, outstanding PRs for this optimization)